amr-wind/CFDSim.H

```diff
@@ -58,6 +63,14 @@ public:
     SimTime& time() { return m_time; }
     const SimTime& time() const { return m_time; }
 
+#ifdef ERF_AW_MULTIBLOCK
+    void set_mbc(MultiBlockContainer* mbc) { m_mbc = mbc; }
```

Comment:
raw pointers make me nervous. Is there some way we could be using unique_ptr instead?

Reply:
Okay thanks for the suggestion.. I'll surely look into this, it's always good to have a more robust code.